### ISSUE TITLE
Dalek Updating & Printing

### ISSUE THREAD

**Issue by klwadland** [marderlab/computers#26]
Leah said that Dalek is having issues with updating. The computer is currently set to auto-update, however, when the update tries to run an error pops up saying there isn't enough space on the disk.

Additionally, when trying to print anything from Dalek the computer won't connect to any printer, and when trying to connect to a previously set up printer, or a new one, an error is thrown saying that it's a dangerous connection hazardous to the computer's health.

**Comment by sg-s** [comment]
yeah don't update it ... let it be. as for the printer issue that's different

**Comment by cosmojg** [comment]
On February 23, 2021 11:46:36 AM EST, klwadland <notifications@github.com> wrote:
>Leah said that Dalek is having issues with updating. The computer is currently set to auto-update, however, when the update tries to run an error pops up saying there isn't enough space on the disk.
>
>Additionally, when trying to print anything from Dalek the computer won't connect to any printer, and when trying to connect to a previously set up printer, or a new one, an error is thrown saying that it's a dangerous connection hazardous to the computer's health.
>

I won't update it since Srinivas said not to, but I'll take a look at the printing issue.

**Comment by cosmojg** [comment]
Alright, the printing issue has been fully resolved! Dalek can now print beautifully from all four printers without any popups. It looks like someone tried to install special printer drivers, and macOS didn't like them for some reason. I simply deleted them, and the issue went away (printing works just fine with the recommended default drivers).

Also, apparently Inky and Blinky had their hostnames mixed up (Inky thought it was Blinky and vice versa) so I fixed that, too.
